core/src/toga/window.py

```diff
@@ -198,6 +199,8 @@ def __init__(
 
         self.on_close = on_close
 
+        self._content = content
```

Comment:
This needs to use the setter, not just set the internal property, as there is other housekeeping that needs to be performed.